providers/aws/services/vpc/vpc_endpoint_services_allowed_principals_trust_boundaries/vpc_endpoint_services_allowed_principals_trust_boundaries_test.py

```diff
@@ -0,0 +1,99 @@
+from unittest import mock
+
+from boto3 import client
+from moto import mock_ec2, mock_elbv2
+
+AWS_REGION = "us-east-1"
+ACCOUNT_ID = "123456789012"
+
+
+class Test_vpc_endpoint_services_allowed_principals_trust_boundaries:
```

Comment:
Also here I'm missing a test using the config values

Reply:
I will create a task for this.